core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
Answer regarding if anyone was using our internals was more of how much impact something like this might cause.  Basically taking that into consideration.  After thorough review, I feel issues were minor but definitely worth catching in a code review so they are not introduced and later expected behavior even if coded in error.  Anyways, all good here.  Thanks.